pkgs/development/compilers/jsonnet/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchFromGitHub, emscripten }:
+
+let version = "0.8.5"; in
+
+stdenv.mkDerivation rec {
```

Comment:
From what I can see you don't need `rec` here, because you have defined `version` outside.